client/config/config.go

```diff
@@ -717,8 +717,24 @@ func DefaultConfig() *Config {
 		NoHostUUID:              true,
 		DisableRemoteExec:       false,
 		TemplateConfig: &ClientTemplateConfig{
```

Comment:
I believe that this is for the client. In case you meant `jobspec`, the only overlap with the `jobspec` template stanza is `wait`. The `jobpsec` value will override the client config as long as it is within the `wait_bounds` if specified. Otherwise it will take the bounded value (one or both depending on if both bounds are crossed).

Reply:
Oh right, sorry, wrong part of the code 😊